apps/web/src/utils/formatEthPrice.ts

```diff
@@ -0,0 +1,22 @@
+import { formatEther, parseEther } from 'viem';
+
+export function formatEthPrice(price?: bigint): number | '...' {
+  if (price === undefined) {
+    return '...';
+  }
+  const value = parseFloat(formatEther(price));
+  if (value < 0.001) {
+    return parseFloat(value.toFixed(4));
+  } else {
+    return parseFloat(value.toFixed(3));
+  }
+}
+
+export function formatWeiPrice(price?: bigint): number | '...' {
+  if (price === undefined) {
+    return '...';
+  }
+
+  const priceInEth = formatEther(price);
+  return parseEther(priceInEth.toString());
+}
```

Comment:
I think these variable names could use some work.
```suggestion
export function weitoEth(wei?: bigint): number | '...' {
  if (wei === undefined) {
    return '...';
  }
  
  return Number(wei / 10n**18n);
}
```

Reply:
refactored per suggestion